testing/conf.lua

```diff
@@ -8,3 +11,14 @@ function love.conf(t)
   t.window.stencil = true
   t.renderers = {"opengl"}
 end
+
+-- custom crash message here to catch anything that might occur with modules 
+-- loading before we hit main.lua
+local function error_printer(msg, layer)
+	print((debug.traceback("Error: " .. tostring(msg), 1+(layer or 1)):gsub("\n[^\n]+$", "")))
+end
+function love.errorhandler(msg)
+	msg = tostring(msg)
+	error_printer(msg, 2)
+  os.exit(1)
```

Comment:
I think this can just return at the end of the function instead of `os.exit`. Calling `os.exit` will stop some things from cleaning up properly, which ends up making OpenAL output a warning message.